pom.xml

```diff
@@ -56,6 +57,24 @@
       <version>${rest-assured.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>folio-service-tools-test</artifactId>
+      <version>${folio-service-tools.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <version>3.1.6</version>
```

Comment:
Why this particular version? Latest is [4.0.2](https://github.com/awaitility/awaitility/releases).

Reply:
updated, thanks